src/ast.hpp

```diff
@@ -226,7 +226,7 @@ struct TypeTerm : public AST {
 };
 
 struct TypeVar : public AST {
```

Comment:
I meant a comment explaining what "typeVar" means

the comment on the next line is probably not necessary